.github/workflows/release.yml

```diff
@@ -93,12 +93,12 @@ jobs:
       - name: "Make"
         if: ${{ success() }}
         run: |
-          PEERD_IMAGE_TAG=${{ env.TAG }} make tests-deps-install ci-aks-ctr
+          PEERD_IMAGE_TAG=${{ env.TAG }} make tests-deps-install ci-aks-llm-pull
 
   ciStreaming:
```

Comment:
Is this enabling AKS image streaming?

Reply:
No, this change is just a refactor of the make target that runs the standard pull CI. The streaming CI is in the `ciStreaming` job in this file.

Reply:
The `azure.sh` script is used to run the CI, enable artifact streaming on the nodepool, etc.